mtest/libmscore/timesig/tst_timesig.cpp

```diff
@@ -303,11 +309,15 @@ void TestTimesig::timesig_78216()
     Measure* m1 = score->firstMeasure();
     Measure* m2 = m1->nextMeasure();
     Measure* m3 = m2->nextMeasure();
+    //Measure* m4 = m3->nextMeasure();
+    //Measure* m5 = m4->nextMeasure();
 
     // verify no timesig exists in segment of final tick of m1, m2, m3
     QVERIFY2(!m1->findSegment(SegmentType::TimeSig, m1->endTick()), "Should be no timesig at the end of measure 1.");
     QVERIFY2(!m2->findSegment(SegmentType::TimeSig, m2->endTick()), "Should be no timesig at the end of measure 2.");
     QVERIFY2(!m3->findSegment(SegmentType::TimeSig, m3->endTick()), "Should be no timesig at the end of measure 3.");
+    //QVERIFY2(!m4->findSegment(SegmentType::TimeSig, m4->endTick()), "Should be no timesig at the end of measure 4.");
+    //QVERIFY2(!m5->findSegment(SegmentType::TimeSig, m5->endTick()), "Should be no timesig at the end of measure 5.");
```

Comment:
Why these lines are commented?

Reply:
Because no such tests (m4 and m5) have yet been written.

Reply:
And I just coppied across what I had for MuseScore 3

Reply:
Forget what I wrote: these are not needed at all

Reply:
removed...